unattended_installer/install_functions/installVariables.sh

```diff
@@ -35,18 +35,18 @@ debug=">> ${logfile} 2>&1"
 ## Offline Installation vars
 readonly BASE_DEST_FOLDER="wazuh-offline"
 readonly WAZUH_DEB_BASE_URL="${BASE_URL}/apt/pool/main/w/wazuh-manager"
-readonly WAZUH_DEB_PACKAGES=( "wazuh-manager_${wazuh_version}-${wazuh_revision}_amd64.deb" )
+readonly WAZUH_DEB_PACKAGES="wazuh-manager_${wazuh_version}-${wazuh_revision}_amd64.deb"
```

Comment:
Instead WAZUH_DEB_PACKAGES it should be MANAGER_DEB_PACKAGE, right?
Also, it could be a good idea to remove the plural of variable names that only have one element.

Reply:
Done: https://github.com/wazuh/wazuh-packages/pull/1465/commits/c77ab55116808ccdcf6cf654fabe4a17aeed1664